src/main/java/org/takes/rq/multipart/RqMtBase.java

```diff
@@ -140,15 +129,15 @@ public RqMtBase(final Request req) throws IOException {
         if (values == null) {
             iter = new VerboseIterable<>(
                 Collections.<Request>emptyList(),
-                new Sprintf(
+                String.format(
```

Comment:
@essobedo `String.format()` will print the string right here, while `Sprintf` will print only when it's needed. in most cases - never.